app/views/schedules/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= content_for(:title) { 'Edit Schedule' } %>
+
+<%= content_for :header do %>
+  <%= render 'shared/breadcrumb_nav' %>
+
+  <h1 class="header__title">Edit schedule</h1>
+
+  <div class="float-end">
+    <button type="submit" class="btn btn-primary" data-form="js-schedule-form">
+      Update
+    </button>
+
+    <%= link_to 'Cancel', pipeline_schedule_path(@pipeline, @schedule), class: 'btn btn-outline-danger' %>
+  </div>
+
+  <div class="clearfix"></div>
+
+<% end %>
+
+<div class='clearfix'></div>
```

Comment:
Do you need a clearfix at the beginning of the template?

Reply:
No we don't need both! It must have been a copy paste haha.